netutils/config/parser.py

```diff
@@ -417,7 +418,7 @@ def _build_multiline_config(self, delimiter):
 class CiscoConfigParser(BaseSpaceConfigParser):
     """Cisco Implementation of ConfigParser Class."""
 
-    regex_banner = re.compile(r"^(banner\s+\S+|\s*vacant-message)\s+(?P<banner_delimiter>\^C|\x03)")
+    regex_banner = re.compile(r"^(banner\s+\S+|\s*vacant-message)\s+(?P<banner_delimiter>\^C|\x03|.{1,1})")
```

Comment:
```suggestion
    regex_banner = re.compile(r"^(banner\s+\S+|\s*vacant-message)\s+(?P<banner_delimiter>\^C|\x03|.{1,1})")
```

I believe this could just be `^\C|.`

Reply:
So use this ? `regex_banner = re.compile(r"^(banner\s+\S+|\s*vacant-message)\s+(?P<banner_delimiter>\^C|.)")`

Reply:
yes, I believe that should yield the same results

Reply:
Yes, same result. Thanks!